definition/ITeam.ts

```diff
@@ -9,6 +9,7 @@ export enum TEAM_TYPE {
 export interface ITeam extends IRocketChatRecord {
 	name: string;
 	type: TEAM_TYPE;
+	roomId?: string;
```

Comment:
Yup, I was having it optional since it should be created after having the actual roomId. But I can have it required using a placeholder value. 

Done!